drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/referral_posting.rb

```diff
@@ -45,16 +45,24 @@ class ReferralPosting < ::HmisExternalApis::HmisExternalApisBase
       referral_result: ::HudUtility.hud_list_map_as_enumerable(:referral_result_map),
     )
 
-    VALID_LOCAL_STATUSES = ['assigned_status', 'accepted_pending_status', 'denied_pending_status'].freeze
-    VALID_LOCAL_STATUS_IDS = statuses.values_at(*VALID_LOCAL_STATUSES).freeze
+    DENIAL_STATUSES = ['assigned_status', 'denied_status', 'denied_pending_status'].freeze
+    ASSIGNED_STATUSES = ['assigned_status', 'accepted_pending_status', 'denied_pending_status'].freeze
 
     validates :status, presence: true
-    validates :status, inclusion: { in: VALID_LOCAL_STATUSES }, on: :hmis_user_action
-    validates :status_note, length: { maximum: 4_000 }, on: :hmis_user_action
-    validates :denial_reason, presence: true, if: :denied_pending_status?, on: :hmis_user_action
-    validates :denial_note, length: { maximum: 2_000 }, on: :hmis_user_action
-    validates :denial_note, presence: true, if: :denied_status?, on: :hmis_user_action
-    validates :referral_result, presence: true, if: :denied_status?, on: :hmis_user_action
+    with_options on: :hmis_user_action do
+      validates :status, inclusion: { in: ASSIGNED_STATUSES }
+      validates :status_note, length: { maximum: 4_000 }
```

Comment:
We should also validate presence of status_note, as that's required by the API. Can you add that in here? It should always be present when updating